drake/examples/Pendulum/runDynamics.cpp

```diff
@@ -1,33 +1,15 @@
 
 #include <iostream>
 #include "Pendulum.h"
-#include "LCMCoordinateFrame.h"
-#include "BotVisualizer.h"
 
 using namespace std;
 
-class PendulumWithBotVis : public Pendulum {
-public:
-  PendulumWithBotVis(const shared_ptr<lcm::LCM>& lcm) : botvis(lcm,"Pendulum.urdf",DrakeJoint::FIXED) {}
-
-  virtual VectorXs output(double t, const VectorXs& x, const VectorXs& u) const override {
-    botvis.output(t,Eigen::VectorXd::Zero(0),x);
-    return Pendulum::output(t,x,u);
-  }
-
-  BotVisualizer botvis;
-};
-
 int main(int argc, char* argv[]) {
   shared_ptr<lcm::LCM> lcm(new lcm::LCM);
   if(!lcm->good())
    return 1;
 
   DrakeSystemPtr p = make_shared<PendulumWithBotVis>(lcm);
```

Comment:
hah.  in fact I had this comment ontop of my runLCM declaration:
// todo: clean this up so it does not take shared ptrs for sys and lcm (this method should not be allowed to maintain ownership)